### ISSUE TITLE
translation bug

### ISSUE THREAD

**Issue by aldisapriansyah** [quran/quran.com-api#385]
When I tried to change the translation to Indonesian, it won't change, please help.

**Comment by moyatvoya** [comment]
It's not only with Indonesian. It doesn't work with any language. Only default language is working

**Comment by aldisapriansyah** [comment]
Yeah, it'll only won't change if we set the page language to other language besides English, but if you use English as the page's language you can choose any translation that you want from the settings. Thanks for responding by the way.
